.gitignore

```diff
@@ -15,3 +15,4 @@ docs/_build
 integration-test/
 tests-env/
 .pytest_cache/
+venv/
```

Comment:
This makes sense to me to add, but as a separate commit from the SSL fix. 

However your commit history is a mess, so normally I'd merge this PR by squashing... but that would conflict with above...